### PR TITLE
culling: enhance quanternion based methods

### PR DESCRIPTION
**Pull request by belom88**

### CODE REVIEW

**Review by ibgreen — APPROVED** [review]

**Review by ibgreen — COMMENTED** [review]
Docs? https://github.com/uber-web/math.gl/blob/master/modules/culling/docs/api-reference/oriented-bounding-box.md

File: modules/culling/src/lib/oriented-bounding-box.d.ts [hunk 1]
```diff
@@ -17,8 +17,14 @@ export default class OrientedBoundingBox {
   // An OrientedBoundingBox of some object is a closed and convex cuboid. It can provide a tighter bounding volume than {@link BoundingSphere} or {@link AxisAlignedBoundingBox} in many cases.
   constructor(center?: readonly number[], halfAxes?: readonly number[]);
 
+  // Returns halfSize array corresponding to I3S OBB halfSize (https://github.com/Esri/i3s-spec/blob/master/docs/1.7/obb.cmn.md)
```
Comment (ibgreen) [inline modules/culling/src/lib/oriented-bounding-box.d.ts:20]: If you use JSDoc style comments, I think might become visible in vscode when hovering:
`/** */`

Comment (belom88) [reply modules/culling/src/lib/oriented-bounding-box.d.ts:20]: Do you mean hypertext? The link works in vscode as is now.

**Comment by belom88** [comment]
> Docs? https://github.com/uber-web/math.gl/blob/master/modules/culling/docs/api-reference/oriented-bounding-box.md

Oh, yes. I'm sorry. I absolutely forgot about it.

**Pull request merged**